src/vcpkg/vcpkgcmdarguments.cpp

```diff
@@ -677,6 +679,13 @@ namespace vcpkg
             Debug::println("Detected CI environment: ", *ci_env);
             get_global_metrics_collector().track_string(StringMetric::DetectedCiEnvironment, *ci_env);
         }
+
+        if (auto gh_repo = gh_repo_env.get())
+        {
+            auto gh_repo_hash = Hash::get_string_hash(*gh_repo, Hash::Algorithm::Sha256);
+            Debug::println("Github repo: ", gh_repo_hash);
+            get_global_metrics_collector().track_string(StringMetric::GithubRepo, gh_repo_hash);
```

Comment:
Probably better to create a `MetricsSubmission` object and call `get_global_metrics_collector()` just once at the end, like it is done in `VcpkgCmdArguments::track_feature_flag_metrics()`.